Frontend/app/app.module.ts

```diff
@@ -1,18 +1,30 @@
 import { NgModule }      from '@angular/core';
```

Comment:
Maybe we should alphabetize these imports or categorize my module, components, etc.